home/templates/home/landingpage.html

```diff
@@ -1,7 +1,44 @@
 {% extends "home/base.html" %}
 {% load static %}
 {% block content %}
+
+<style type="text/css">
+        body{
+         background: url('{% static "home/background.png" %}') no-repeat center center fixed;
+         -webkit-background-size: cover;
+         -moz-background-size: cover;
+         -o-background-size: cover;
```

Comment:
Maybe create a folder in the static called CSS where you can make CSSfile and put all the style there, as suggested by Barak

Reply:
I added .